source/index.html

```diff
@@ -51,6 +52,7 @@ <h1 class="selection__title">
 
     <section class="offer">
       <div class="offer__wrapper">
+        <div class="offer__inner">
         <h2 class="offer__title">
           Похудение
         </h2>
```

Comment:
Если добавляем обертку, то и отступы необходимо править для тех элементов, что оказываются в ней
```suggestion
        <div class="offer__inner">
          <h2 class="offer__title">
            Похудение
          </h2>
```

Reply:
Ага подправила